app/src/test/kotlin/com/wire/android/ui/home/conversations/ConversationsViewModelTest.kt

```diff
@@ -556,7 +556,7 @@ class ConversationsViewModelTest {
         content = Text("Some Text Message"),
         conversationId = ConversationId("convo-id", "convo.domain"),
         date = "some-date",
-        senderUserId = senderId,
+        sendeUserId = senderId,
```

Comment:
```suggestion
        senderUserId = senderId,
```
I think there is typo here, and that's why the tests are failing 🤔